python/semantic_kernel/connectors/memory/azure_cosmosdb/azure_cosmosdb_settings.py

```diff
@@ -21,4 +21,4 @@ class AzureCosmosDBSettings(KernelBaseSettings):
     env_prefix: ClassVar[str] = "COSMOSDB_"
 
     api: str | None = None
-    connection_string: SecretStr | None = Field(None, alias="AZCOSMOS_CONNSTR")
```

Comment:
Thanks for having a look at this! This would be a breaking change, I think we need to add a additional setting and then have a model post validator that syncs the values!

Reply:
It is and it was forgotten for a long time so it will break the code for anyone using AZCOSMOS_CONNSTR
Yet the original class functionality it not working correctly i.e. you can't pass a connection_string param. 

I suggested the other solution on the issue which will allow for both to work although I don't prefer it to conform with the other classes.

i will modify the pr to allow for both the alias and the original class attribute